socorro/unittest/external/boto/test_crash_data.py

```diff
@@ -0,0 +1,156 @@
+import json
+
+import mock
+from nose.tools import eq_, assert_raises
+from boto.exception import StorageResponseError
+
+from configman import ConfigurationManager
+
+from socorrolib.lib import MissingArgumentError
+from socorro.external.boto.crash_data import SimplifiedCrashData
+from socorro.external.crashstorage_base import CrashIDNotFound
+from socorro.unittest.testbase import TestCase
+
+
+class SimplifiedCrashData(TestCase):
+
+    def _get_config(self, sources, extra_values=None):
+        self.mock_logging = mock.Mock()
+
+        config_definitions = []
+        for source in sources:
+            conf = source.get_required_config()
+            conf.add_option('logger', default=self.mock_logging)
+            config_definitions.append(conf)
+
+        values_source = {'logger': self.mock_logging}
+
+        config_manager = ConfigurationManager(
+            config_definitions,
+            app_name='testapp',
+            app_version='1.0',
+            app_description='',
+            values_source_list=[values_source],
+            argv_source=[],
+        )
+
+        return config_manager.get_config()
+
+    def get_s3_store(self, storage_class=SimplifiedCrashData):
```

Comment:
I totally agree with that principle!
What tripped me up was that it got confused with what `SimplifiedCrashData` was. There's the implementation class and this test class. So I renamed this test class to `TestSimplifiedCrashData`.